src/main/java/org/apache/lucene/analysis/kr/utils/DictionaryUtil.java

```diff
@@ -312,12 +296,12 @@ public static String combineAndEomiCheck(char s, String eomi) throws MorphExcept
 	 * @param type	1: josa, 2: eomi
 	 * @throws MorphException
```

Comment:
Wait are you replying to the "no longer true" because that's about the javadoc being outdated.